client/components/token-field/test/index.jsx

```diff
@@ -301,21 +292,22 @@ describe( 'TokenField', function() {
 			expect( getSelectedSuggestion() ).to.deep.equal( expectedSuggestion );
 
 			function testSavedState( isActive ) {
-				expect( wrapper.state.tokens ).to.deep.equal( expectedTokens );
-				expect( textInputNode.value ).to.equal( '' );
+				expect( wrapper.state( 'tokens' ) ).to.deep.equal( expectedTokens );
+				expect( textInputNode.props().value ).to.equal( '' );
 				expect( getSelectedSuggestion() ).to.equal( null );
-				expect( wrapper.refs.tokenField.state.isActive ).to.equal( isActive );
+				expect( tokenFieldNode.find( 'div' ).first().hasClass( 'is-active' ) ).to.equal( isActive );
 			}
 
 			document.activeElement = document.body;
-			TestUtils.Simulate.blur( textInputNode );
-			setTimeout( function() {
-				// After blur, need to wait for TokenField#_blurTimeoutID
+			textInputNode.simulate( 'blur' );
+
+			// After blur, need to wait for TokenField#_blurTimeoutID
+			defer( function() {
 				testSavedState( false );
-				TestUtils.Simulate.focus( textInputNode );
+				textInputNode.simulate( 'focus' );
 				testSavedState( true );
 				done();
-			}, 10 );
```

Comment:
`defer` is the [equivalent of `setTimeout( fn, 1 );`](https://github.com/lodash/lodash/blob/4.10.0/lodash.js#L9467), so not quite the same as was here previously. Not sure whether the 10 millisecond delay was chosen for any particular reason (cc @nylen), but would be happy to see it reduced if possible.